### PR TITLE
1.7.0-beta4

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
Preparing a new beta to address issues in last release and to add some rebase love.

 - [x] bump version
 - [x] draft release notes
 - [x] approve notes
 - [x] publish release

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "1.7.0-beta4": [
+      "[Fixed] Ensure manual resolution choices are in sync with rebase flow - #7255",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Manual conflict resolution choice not updated when resolving rebase conflicts - #7255",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+      "[Fixed] Show \"force push\" label after rebase which requires it - #4976 #7138 #7231"
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Menu items don't display the expected verbiage for force push and removing a repository - #4976 #7138"
```

Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Fixed] Show \"force push\" in menu after rebase completed and requires it - #4976 #7138 #7231"
```

Comment (shiftkey) [reply changelog.json]: @iAmWillShepherd were we going to take in this change?

**Review by shiftkey — APPROVED** [review]

**Review by billygriffin — APPROVED** [review]

**Pull request merged**